### PR TITLE
remove incorrect wording and fix typos

### PR DESCRIPTION
**Pull request by alexthomasTG**

### CODE REVIEW

File: modules/querying/pages/accumulators.adoc [hunk 1]
```diff
@@ -788,7 +788,7 @@ The element type can be any base type or tuple.
 For SetAccum, the `+= arg` operation adds a non-duplicate element or set of elements to the set.
 If an element is already represented in the set, then the SetAccum state does not change.
 
-SetAccum also can be used with the three canonical set operators: `UNION`, `INTERSECT`, and `MINUS` (see the xref:operators-and-expressions.adoc#_setbag_expression_and_operators[Set/Bag Expression and Operators] section for more details).
+SetAccum also can be used with the three canonical set operators: `UNION`, `INTERSECT`, and `MINUS` (see the xref:operators-and-expressions.adoc#setbag-expression-and-operators[Set/Bag Expression and Operators] section for more details).
```
Comment (lennessyy) [inline modules/querying/pages/accumulators.adoc]: are you sure there is no preceding underscore?

File: modules/querying/pages/accumulators.adoc [hunk 2]
```diff
@@ -1447,7 +1447,7 @@ MapAccum<SetAccum<INT>, INT> # illegal
 GroupByAccum<INT a, STRING b, MaxAccum<INT> maxs, ListAccum<ListAccum<INT>> lists>
 ----
 
-. ArrayAccum: Unlike the other accumulators in this list, where nesting is optional, nesting is mandatory for ArrayAccum. See the xref:querying:accumulators.adoc#_arrayaccum[ArrayAccum] section.
+. ArrayAccum: Unlike the other accumulators in this list, where nesting is optional, nesting is mandatory for ArrayAccum. See the xref:querying:accumulators.adoc#arrayaccum[ArrayAccum] section.
```
Comment (lennessyy) [inline modules/querying/pages/accumulators.adoc]: https://docs.tigergraph.com/gsql-ref/current/querying/accumulators#_arrayaccum is the link for array accum - by default Antora generates the anchors with a preceding underscore.

**Review by lennessyy — APPROVED** [review]
Alex accidentally edited Lenny's comment - it said to put the preceding underscores back

File: modules/querying/pages/accumulators.adoc (hunk 2 above)
Comment (alexthomasTG) [reply modules/querying/pages/accumulators.adoc]: Weird, they showed up as typos/not found for me in Intellij

**Pull request merged**